extras/check_code_style.sh

```diff
@@ -232,7 +239,13 @@ for f in $file_names; do
     *.py )
       echo "Check PEP8 on file $f:"
 
-      if ! pep8_result=`pep8 --first --ignore=$PEP8_IGNORES $f` ; then
+      if [[ $f =~ $EXAMPLE_DIRS ]]; then
```

Comment:
@heplesser  I just checked it -- https://github.com/PyCQA/pycodestyle/pull/27#issuecomment-218397241 ; Not possible, and apparently will never be possible -- it's a "philosophical" problem until someone forks pep8. The kind of philosophical problems that tend to crop up particularly often in the python community.